Server/API/rooms/getRoomById.go

```diff
@@ -0,0 +1,28 @@
+package rooms
+
+import (
+	"context"
+	"log"
+	"net/http"
+	"time"
+
+	"github.com/labstack/echo/v4"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+	"ishtaloo.io/DB/Collections"
+	entities "ishtaloo.io/Entities"
+)
+
+func getRoomById(c echo.Context) error {
+	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
+	defer cancel()
+	roomsCollection := Collections.RoomsGetCollection()
+	roomId := c.Param("roomId")
+	id, _ := primitive.ObjectIDFromHex(roomId)
```

Comment:
What if roomId is invalid?

Reply:
I didn't put any protection yet, because it's only the demo, so I thought to just add it later, do you agree?

Reply:
Cool, big like